src/visualization/types/SimpleTable/PagedTable.tsx

```diff
@@ -281,20 +281,18 @@ const PagedTable: FC<Props> = ({result, properties}) => {
   }, [size])
 
   useEffect(() => {
-    if (size > maxSize) {
-      setMaxSize(size)
-    }
-  }, [size])
+    setMaxSize(measurePage(result, 0, height))
+  }, [height, result])
 
   useEffect(() => {
     setPage(1)
   }, [result])
 
   useEffect(() => {
-    if (size) {
-      setTotalPages(Math.ceil((result?.table?.length ?? 0) / size))
+    if (maxSize) {
+      setTotalPages(Math.ceil((result?.table?.length ?? 0) / maxSize))
     }
-  }, [height, result])
+  }, [maxSize])
```

Comment:
Nice work! 🔥 

However, I see lint warnings for the hooks that you have updated. To be fair, the lint warnings were already there. Are we just going to ignore the lint warnings? There's a way to disable the lint warnings if we don't want them there. Add a comment add the end of each line of the dependency array that disables the lint rule for that line.

See here for examples:
<img width="615" alt="Screen Shot 2021-08-04 at 3 46 15 PM" src="https://user-images.githubusercontent.com/10736577/128265187-0b646fb0-8786-466c-ad9e-7993da091849.png">